pkg/process/cloudlet_services.go

```diff
@@ -269,6 +281,51 @@ func StopCRMService(ctx context.Context, cloudlet *edgeproto.Cloudlet, haRole HA
 	return nil
 }
 
+var maxPrimaryCrmStartupWait = 10 * time.Second
+
+func StartCRMServicesHA(ctx context.Context, cloudlet *edgeproto.Cloudlet, pfConfig *edgeproto.PlatformConfig, redisCfg *rediscache.RedisConfig, crmOps ...CrmServiceOp) error {
```

Comment:
We never test this on a real platform. Do you think we actually still need HA CRMs? If we are moving to CCRM providing HA, this feels like throwaway code, or something that will just accumulate technical debt.

Reply:
There is still the restricted cloudlet case, where we would need it if HA is needed. I agree it's not well tested and not likely to be used anyway, but (like some other features :) I didn't want to rip it out just because it's inconvenient. I think if we discuss and agree there's no place for it (i.e. we don't support HA for restricted cloudlets, which could be reasonable), then we can rip it out. Let's discuss.